packages/core/src/controllers/BlockchainApiController.ts

```diff
@@ -242,6 +245,40 @@ export const BlockchainApiController = {
     })
   },
 
+  async lookupEnsName(name: string) {
+    return api.get<BlockchainApiLookupEnsName>({
+      path: `/v1/profile/account/${name}.wcn.id?projectId=${OptionsController.state.projectId}`
+    })
+  },
+
+  async reverseLookupEnsName({ address }: { address: string }) {
+    // Should add /coinType subroute once ready in BlockchainApi
```

Comment:
Should be removed as confirmed https://walletconnect.slack.com/archives/C06L6HUN1H6/p1715937712951339?thread_ts=1715932794.012259&cid=C06L6HUN1H6